gw-rss-aggregator/src/main/java/ru/otus/storage/StorageImpl.java

```diff
@@ -0,0 +1,99 @@
+package ru.otus.storage;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.dizitart.no2.Nitrite;
+import org.dizitart.no2.objects.ObjectRepository;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import static org.dizitart.no2.objects.filters.ObjectFilters.and;
+import static org.dizitart.no2.objects.filters.ObjectFilters.eq;
+
+import ru.otus.model.Feed;
+import ru.otus.model.Message;
+
+@Component
+@EnableConfigurationProperties(StorageProperties.class)
+public class StorageImpl implements Storage {
```

Comment:
```suggestion
public class NitriteStorage implements Storage {
```

Хотя по набору методов это больше похоже на какой-то Service, а не на Storage.